src/elements/content-sidebar/__tests__/ActivitySidebar.test.js

```diff
@@ -763,6 +764,26 @@ describe('elements/content-sidebar/ActivitySidebar', () => {
         });
     });
 
+    describe('handleAnnotationEdit()', () => {
+        test('should call updateAnnotation API', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+            instance.fetchFeedItems = jest.fn();
+
+            wrapper.instance().handleAnnotationEdit({
+                id: '123',
+                text: 'hello',
+                permissions: {
+                    can_edit: true,
+                    can_delete: true,
+                },
+            });
+
+            expect(api.getFeedAPI().updateAnnotation).toBeCalled();
+            expect(instance.fetchFeedItems).toHaveBeenCalled();
```

Comment:
We try to should stay consistent with using either `toBeCalled` or `toHaveBeenCalled`, since the latter is an alias of the former. What does the codebase use most often?

Reply:
It looks like the codebase has around 3.3k instances of `toHaveBeenCalled` whereas `toBeCalled` has around 1.3k. I'll go with `toHaveBeenCalled`.